src/test/java/org/cloudfoundry/promregator/fetcher/MetricsFetcherTest.java

```diff
@@ -1,198 +1,198 @@
-package org.cloudfoundry.promregator.fetcher;
```

Comment:
Warning: Converted newlines into crlf (from lf) again

Reply:
Yes , Fixed in subsequent.. Seems my git client was not setup correctly. I belive its all sorted out now